app/api/helpers/mail.py

```diff
@@ -374,8 +374,8 @@ def send_email_to_attendees(order, purchaser_id, attachments=None):
             )
         else:
             # The Ticket holder is not the purchaser
-            start_date = "".join(str(order.event.starts_at.date()).split("-"))
-            end_date = "".join(str(order.event.ends_at.date()).split("-"))
+            start_date = (order.event.starts_at.date()).strftime('%Y%m%d')
+            end_date = (order.event.ends_at.date()).strftime('%Y%m%d')
```

Comment:
You need to convert the time to event timezone

Reply:
done